src/Tasks/CheckJobHealthTask.php

```diff
@@ -48,11 +48,11 @@ public function run($request)
             $unhealthyJobCount = $unhealthyJobCount + $count;
         }
 
-        if ($unhealthyJobCount === 0) {
-            echo 'All jobs are healthy';
-        } else {
-            die(1);
+        if ($unhealthyJobCount > 0) {
+            throw new Exception("$unhealthyJobCount jobs are unhealthy");
```

Comment:
How about a more specific exception? We could create an exception like "UnhealthJobsException".

Reply:
Maybe that should be another PR as the module is still using `Exception` in 6 other places?

```
$ grep "use Exception;" .
./tests/QueuedJobsTest/TestExceptingJob.php:7:use Exception;
./src/Tasks/CheckJobHealthTask.php:5:use Exception;
./src/Tasks/PublishItemsTask.php:5:use Exception;
./src/Jobs/GenerateGoogleSitemapJob.php:5:use Exception;
./src/Services/QueuedJobService.php:7:use Exception;
./src/Services/JobErrorHandler.php:5:use Exception;
```